README.md

```diff
@@ -55,13 +56,15 @@ To run the project click on the HTML document on your local machine or use the l
 
 - solid Linting  
 - responsive design 
+- Clear message 
+- best illustration  
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-The live demo link is currently unavailable due to technical stuff and maintenance, please be patient as we are working on something presentable 
+The live demo link is currently unavailable due to technical stuff and maintenance, please be patient as we are working on something presentable  
```

Comment:
It is important to follow the project requirements by deploying the website to make sure it is available for viewing